functions/nodejs-apps.bash

```diff
@@ -342,3 +342,60 @@ zigbee2mqtt_setup() {
     whiptail --title "Operation successful" --msgbox "$installSuccessText" 15 80
   fi
 }
+
+## Function for installing a npm package for the JS Scripting Automation Add-On
+##
+##    jsscripting_npm_install(String packageName, String mode)
+##    Available values for mode: "update", "install". Defaults to "install".
+##
+jsscripting_npm_install() {
+  if [ "${1}" == "" ]; then echo "FAILED. Provide packageName."; return 1; fi
```

Comment:
Please check for $UNATTENDED and distinguish between interactive install (chosen via menu) and initial (unattended) install.
If interactive, provide feedback via whiptail info boxes.
If unattended, be verbose about what you're doing
(like echo "Installing foobar ...\c" ..... echo "OK" ..... )

See how installationb is done for other packages/function for examples what I mean by that.

Apply to uninstalling, too.